2025-10-20 16:53:37,690 ERROR [LDA.LDAPAuthenticationService] (executor-thread-1) Error contacting LDAP server ldap://oa.bbl: Unable to connect

Exception in LDAPAuthenticationService.java:57 ''), [:] for the terminal, [h] for more options>
          55                                                                                                                                                                                                                        
          56          try {                                                                                                                                                                                                         
        ? 57              DirContext ctx = new InitialDirContext(new Hashtable<>(environment));                                                                                                                                     
          58              ctx.close();                                                                                                                                                                                              
          59              log.info("LDAPAuthenticationService->validateUserCredential: {} - Success", username);                                                                                                                    
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
Exception in LoginResource.java:75                                                                                                                                                                                                  
          73                                                                                                                                                                                                                        
          74              try {                                                                                                                                                                                                     
        ? 75                  boolean ldapAuthenticated = ldapAuthenticationService.validateUserCredentials(request.email, request.password);                                                                                       
          76                  if (ldapAuthenticated) {                                                                                                                                                                              
          77                      log.info(username);                                                                                                                                                                               
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
Exception in LDAPAuthenticationService.java:57                                                                                                                                                                                      
          55                                                                                                                                                                                                                        
          56          try {                                                                                                                                                                                                         
        ? 57              DirContext ctx = new InitialDirContext(new Hashtable<>(environment));                                                                                                                                     
          58              ctx.close();                                                                                                                                                                                              
          59              log.info("LDAPAuthenticationService->validateUserCredential: {} - Success", username);                                                                                                                    
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                    
Exception in LoginResource.java:75                                                                                                                                                                                                  
          73                                                                                                                                                                                                                        
          74              try {                                                                                                                                                                                                     
        ? 75                  boolean ldapAuthenticated = ldapAuthenticationService.validateUserCredentials(request.email, request.password);                                                                                       
          76                  if (ldapAuthenticated) {                                                                                                                                                                              
          77                      log.info(username);                                                                                                                                                                               
                                                                                                                                                                                                                                    
: javax.naming.CommunicationException: oa.bbl:389 [Root exception is java.net.SocketTimeoutException: Connect timed out]                                                                                                            
        at java.naming/com.sun.jndi.ldap.Connection.<init>(Connection.java:253)                                                                                                                                                     
        at java.naming/com.sun.jndi.ldap.LdapClient.<init>(LdapClient.java:141)                                                                                                                                                     
        at java.naming/com.sun.jndi.ldap.LdapClient.getInstance(LdapClient.java:1620)                                                                                                                                               
        at java.naming/com.sun.jndi.ldap.LdapCtx.connect(LdapCtx.java:2848)                                                                                                                                                         
        at java.naming/com.sun.jndi.ldap.LdapCtx.<init>(LdapCtx.java:349)                                                                                                                                                           
        at java.naming/com.sun.jndi.ldap.LdapCtxFactory.getLdapCtxFromUrl(LdapCtxFactory.java:229)                                                                                                                                  
        at java.naming/com.sun.jndi.ldap.LdapCtxFactory.getUsingURL(LdapCtxFactory.java:189)                                                                                                                                        
        at java.naming/com.sun.jndi.ldap.LdapCtxFactory.getUsingURLs(LdapCtxFactory.java:247)                                                                                                                                       
        at java.naming/com.sun.jndi.ldap.LdapCtxFactory.getLdapCtxInstance(LdapCtxFactory.java:154)                                                                                                                                 
        at java.naming/com.sun.jndi.ldap.LdapCtxFactory.getInitialContext(LdapCtxFactory.java:84)                                                                                                                                   
        at java.naming/javax.naming.spi.NamingManager.getInitialContext(NamingManager.java:520)                                                                                                                                     
        at java.naming/javax.naming.InitialContext.getDefaultInitCtx(InitialContext.java:305)                                                                                                                                       
        at java.naming/javax.naming.InitialContext.init(InitialContext.java:236)                                                                                                                                                    
        at java.naming/javax.naming.InitialContext.<init>(InitialContext.java:208)                                                                                                                                                  
        at java.naming/javax.naming.directory.InitialDirContext.<init>(InitialDirContext.java:130)                                                                                                                                  
        at LDAPServices.LDAPAuthenticationService.validateUserCredentials(LDAPAuthenticationService.java:57)                                                                                                                        
        at LDAPServices.LDAPAuthenticationService_ClientProxy.validateUserCredentials(Unknown Source)                                                                                                                               
        at Testing.LoginResource.login(LoginResource.java:75)                                                                                                                                                                       
        at Testing.LoginResource_Subclass.login$$superforward(Unknown Source)                                                                                                                                                       
        at Testing.LoginResource_Subclass$$function$$1.apply(Unknown Source)                                                                                                                                                        
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)                                                                                                                         
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)                                                                                                                         
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:136)                                                                                            
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:107)                                                                                            
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired.doIntercept(TransactionalInterceptorRequired.java:38)                                                                                       
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.intercept(TransactionalInterceptorBase.java:61)                                                                                                 
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired.intercept(TransactionalInterceptorRequired.java:32)                                                                                         
        at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired_Bean.intercept(Unknown Source)                                                                                                              
        at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)                                                                                                                                          
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)                                                                                                                         
        at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)                                                                                                                                   
        at Testing.LoginResource_Subclass.login(Unknown Source)                                                                                                                                                                     
        at Testing.LoginResource$quarkusrestinvoker$login_ac7b04d13117f22f5c6217598ed82eb24eb329f4.invoke(Unknown Source)                                                                                                           
        at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)                                                                                                                          
        at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)                                                                          
        at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)                                                                                                    
        at io.quarkus.vertx.core.runtime.VertxCoreRecorder$15.runWith(VertxCoreRecorder.java:637)                                                                                                                                   
        at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)                                                                                                                                  
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)                                                                                                                                        
        at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)                                                                                                                                   
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)                                                                                                                                  
        at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)                                                                                                                                                     
        at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)                                                                                                                                 
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)                                                                                                                                    
        at java.base/java.lang.Thread.run(Thread.java:1583)                                                                                                                                                                         
Caused by: java.net.SocketTimeoutException: Connect timed out                                                                                                                                                                       
        at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)                                                                                                                                            
        at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)                                                                                                                                                       
        at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)                                                                                                                                                     
        at java.base/java.net.Socket.connect(Socket.java:751)                                                                                                                                                                       
        at java.naming/com.sun.jndi.ldap.Connection.createConnectionSocket(Connection.java:333)                                                                                                                                     
        at java.naming/com.sun.jndi.ldap.Connection.createSocket(Connection.java:285)                                                                                                                                               
        at java.naming/com.sun.jndi.ldap.Connection.<init>(Connection.java:232)        
